x-pack/plugins/cases/public/common/use_cases_toast.tsx

```diff
@@ -34,6 +34,20 @@ const EuiTextStyled = styled(EuiText)`
   `}
 `;
 
+function getAlertsCount(attachments: CaseAttachments): number {
+  let alertsCount = 0;
+  for (const attachment of attachments) {
+    if (attachment.type === CommentType.alert) {
+      if (Array.isArray(attachment.alertId) && attachment.alertId.length > 1) {
+        alertsCount += attachment.alertId.length;
+      }
+
+      alertsCount++;
```

Comment:
This is a bit confusing. It took me some time to realize that this also counts arrays with one item. Maybe a comment will help.

Reply:
Also, arrays of `length > 1` will count one more time. For example:

```
getAlertsCount([{type: 'alert', alertId: ['1', '2']}])
```

returns `3` but it should return `2`.

Reply:
Another suggestion would be to convert the function to

```
function hasMultipleAlerts(attachments) {
  let alertsCount = 0;
  for (const attachment of attachments) {
    if (attachment.type === 'alert') {
      if (Array.isArray(attachment.alertId) && attachment.alertId.length > 1) {
        return true
      }

      alertsCount++;
      
      if (alertsCount > 1) {
          return true
      }
    }
  }
  return false
}
```

Reply:
I'll keep the counting one. I fixed the logic to fix the extra count.

Reply:
I also added a comment to clarify the logic.